package.json

```diff
@@ -8,8 +8,11 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@popperjs/core": "^2.9.2",
+    "bootstrap": "^5.0.0",
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
+    "vue-fixed-header": "^3.2.15",
```

Comment:
It seems like you're not using this, but using the bootstrap header instead? If thats true, remove this package.